### ISSUE TITLE
run_amd   error: could not convert string to float: '.'

### ISSUE THREAD

**Issue by schaedeli** [pharmpy/pharmpy#948]
Hi,
I have tried to start the run_amd() tool with settings close to the ones in the Pharmpy documentation example. This returned an error message, apparently without starting the tool. See below. 
Thanks, Franziska

Here is my code, that I tried:
```
#define initial estimates
MAT = 0.5  # 1/KA <- if oral
CL  = 6.0  
V   = 750    #
sp1 <- "TRANSITS([1,3],*);PERIPHERALS(1..2);LAGTIME()"
order1 <- c('structural', 'iivsearch', 'residual', 'allometry', 'covariates')

res <- run_amd(input=dataset_path,
               modeltype='pk_oral', mat_init = MAT, vc_init = V, cl_init = CL,
               order=order1,
               categorical=c('SEX'),
               continuous=c('AGE'),
               allometric_variable='BWT',
               search_space = sp1)
```
The error message and traceback is here:
```
 Error in py_call_impl(callable, dots$args, dots$keywords) : 
  ValueError: could not convert string to float: '.' 
3.
stop(structure(list(message = "ValueError: could not convert string to float: '.'", 
    call = py_call_impl(callable, dots$args, dots$keywords), 
    cppstack = NULL), class = c("Rcpp::exception", "C++Error", 
"error", "condition"))) 
2.
pharmpy$tools$run_amd(input, modeltype, cl_init, vc_init, mat_init, 
    search_space, lloq, order, categorical, continuous, allometric_variable, 
    occasion) 
1.
run_amd(input = dataset_path, modeltype = "pk_oral", mat_init = MAT, 
    vc_init = V, cl_init = CL, order = order1, categorical = c("SEX"), 
    continuous = c("AGE"), allometric_variable = "BWT", search_space = sp1) 

```
It is a code that I previously tried with the same model data set, with `run_modelsearch()` that ran fine.

I have also tried this code, but the error message was the same. Any idea what is the issue?
```
order2 <- c('structural')
> res <- run_amd(input=dataset_path,
+                modeltype='pk_oral', mat_init = MAT, vc_init = V, cl_init = CL,
+                order=order2, 
+                search_space = sp1)
```

**Comment by stellabelin** [comment]
This issue is (I think) due to the fact that AMD doesn't see this as a NONMEM dataset but rather as a regular .csv. When you use `run_modelsearch()`, since the input is a NONMEM model Pharmpy also does some processing when reading in the dataset, i.e. replacing `.` with `0`. We are working on a solution, but a workaround would be that you simply replace the `.` with `0` before running the tool.

**Comment by stellabelin** [comment]
This was fixed in 5af21121584199d96c0ebeaeacedf61b63a88d4b, closing the issue.
